### ISSUE TITLE
Begginer in api

### ISSUE THREAD

**Issue by Abhicoder03** [vedangj044/just-calling-random-api#3]
Hi, I want to learn about api. I just want you to assign me a basic task of api under hacktoberfest 2021. In this way i can learn about api as well as under hacktoberfest. Could you please do this.

**Comment by vedangj044** [comment]
Hello, Yeah this project can help you get started with APIs. In this project you can choose any API and make a contribution based on it. So, it's better than me assigning you a task. Go to https://github.com/public-apis/public-apis and choose any API. Once you make a choose, call it in your react component.

Detailed instruction are mentioned in the README, you can ping me on Gitter or discord.

**Comment by juhiechandra** [comment]
What sort of APIs should I add? When should I make PR?

**Comment by vedangj044** [comment]
You should open a PR in October. You can add any API. It should be something you would like to show us. Like random Advice API, random cat pictures API, etc. Once you contribute it would be visible on the main page ( https://vedangj044.github.io/just-calling-random-api/ ), so choose something which you would like people to see on the main page. You can find a list of APIs you can use here - https://github.com/public-apis/public-apis

**Comment by sudo-nick16** [comment]
> You should open a PR in October. You can add any API. It should be something you would like to show us. Like random Advice API, random cat pictures API, etc. Once you contribute it would be visible on the main page ( https://vedangj044.github.io/just-calling-random-api/ ), so choose something which you would like people to see on the main page. You can find a list of APIs you can use here - https://github.com/public-apis/public-apis

Can we use APIs other than the ones listed on "https://github.com/public-apis/public-apis"?

**Comment by vedangj044** [comment]
Sure, any API would work, you can even create your own API. 

But keep in mind that you should have proper rights to use it and it must be HTTPS.

**Comment by varunsh17** [comment]
Will you be making more issues coz I also wanted to contribute , as I have learned API and want to test over ..

**Comment by yash37158** [comment]
Do we need to be assigned for working with APIs, or anyone can raise their PR.

**Comment by vedangj044** [comment]
Anyone can raise a PR. :tada:

**Comment by anjaligoswami** [comment]
Hello, @vedangj044 I would like to work on this issue.

**Comment by vedangj044** [comment]
Sure @anjaligoswami, there is no issue in particular. All you have to do is call a public API In your react component and contribute. I would ask you to go through the README, it has step-wise instruction on how to contribute.

**Comment by N-Deepika** [comment]
Hey! Can I take this issue?

**Comment by vedangj044** [comment]
@N-Deepika Yes absolutely. You are free to choose any API you like, and use it in your react component. Example - I chose the Random Quotes API, so I created a new component that calls the Random Quote API and displayed the response. Once done I created a Pull Request. 

You can do the same with some other API. You will find step-by-step instructions on how to contribute in the README ( https://github.com/vedangj044/just-calling-random-api/blob/main/README.md )

**Comment by MuditWadhwa** [comment]
Hi, I would like to contribute to this repository as well by taking up this issue.

**Comment by vedangj044** [comment]
@MuditWadhwa awesome. You can contribute by creating a component and calling any public API through it. This issue is just to guide new comers. I would suggest you to go through the README ( https://github.com/vedangj044/just-calling-random-api/blob/main/README.md ), where you will find step wise instructions on how to make a PR and complete hacktoberfest.

**Comment by sterlingSid04** [comment]
Hello, I would like to contribute to this repository as well.

**Comment by vedangj044** [comment]
@sterlingSid04 awesome. You can contribute by creating a component and calling any public API through it. I would suggest you go through the README ( https://github.com/vedangj044/just-calling-random-api/blob/main/README.md ), where you will find stepwise instructions on how to make a PR and complete hacktoberfest.

**Comment by baycak** [comment]
Hi @vedangj044 
I create a [PR](#103) for movie list api

Thank you
